tracing/tracer_test.go

```diff
@@ -67,34 +62,33 @@ func TestInitGlobalTracer_Disabled(t *testing.T) {
 }
 
 func TestInitGlobalTracer_NoServiceName(t *testing.T) {
-	tracer, closer, err := InitGlobalTracer(_emptyJaegerConfig, "", getLogger(), _statsReporter)
+	tracer, closer, err := InitGlobalTracer(_emptyJaegerConfig, "", zap.L(), _statsReporter)
 	assert.Error(t, err)
 	assert.Nil(t, tracer)
 	assert.Nil(t, closer)
 }
 
 func TestLoadAppConfig(t *testing.T) {
-	jConfig := loadAppConfig(_emptyJaegerConfig, getLogger())
+	jConfig := loadAppConfig(_emptyJaegerConfig, zap.L())
 	assert.NotNil(t, jConfig)
 	assert.NotNil(t, jConfig.Logger)
 }
 
 func TestLoadAppConfig_JaegerConfigWithLogger(t *testing.T) {
-	jConfig := loadAppConfig(_jaegerConfigWithLogger, getLogger())
+	jConfig := loadAppConfig(_jaegerConfigWithLogger, zap.L())
 	assert.NotNil(t, jConfig)
 	assert.Equal(t, jaeger.NullLogger, jConfig.Logger)
 }
 
 func TestLoadAppConfig_NilJaegerConfig(t *testing.T) {
-	jConfig := loadAppConfig(nil, getLogger())
+	jConfig := loadAppConfig(nil, zap.L())
```

Comment:
Yep, these are extremely short by design. Many logging packages have top-level `Error`, `Infof`, etc. functions; I really, really don't want these in zap, which means that we need a minimally noisy way to get the global logger. `zap.GlobalLogger().Info("something")` seems like a lot; `zap.L().Info("something")` is much less intrusive.

This is definitely a matter of taste, though - file an issue and we can discuss pros and cons there.